ast.d

```diff
@@ -148,6 +150,30 @@ OperatorType parseOperatorType(string input)
     }
 }
 
+enum OperatorClass
+{
+    Math,
+    Relational,
+    Logical
+}
```

Comment:
D uses ```~``` for string/array concatenation, which is pretty nice for being more clear about what operation is happening.